drivers/hud_pit/app/models/hud_pit/generators/pit/fy2025/base.rb

```diff
@@ -292,15 +299,29 @@ def services_on_pit_date(batch)
         },
         children_of_youth_parents: {
           title: 'Total Children in Parenting Youth Households',
-          query: a_t[:head_of_household].eq(false).and(child_clause),
+          query: [
+            child_of_hoh_clause,
+            child_clause,
+          ].reduce(&:and),
         },
         children_of_18_to_24_parents: {
+          # https://www.hudexchange.info/faqs/3470/how-should-two-parent-parenting-youth-households-be-reported-in-the-pit/
+          # count children under oldest parent (aged 18-24)
           title: 'Children in households with parenting youth age 18 to 24 (children under age 18 with parents age 18 to 24)',
-          query: a_t[:head_of_household].eq(false).and(child_clause).and(a_t[:hoh_age].in(18..24)),
```

Comment:
I think it's more correct to check the max age of any adult given the HUD guidance. Or perhaps we should check the max age of household members who are HOH or spouse of HOH